alerter.c

```diff
@@ -24,8 +34,11 @@ void alertInCelcius(float farenheit) {
 }
 
 int main() {
+    alertInCelcius(32);
+    alertInCelcius(-303.6);
     alertInCelcius(400.5);
     alertInCelcius(303.6);
+    assert(alertFailureCount!=0);

```

Comment:
This assert will pass for any non-zero alertFailureCount (e.g., even if it alerts when not needed). Try making the assert(s) more specific.